tests/search.spec.ts

```diff
@@ -0,0 +1,37 @@
+import test, { expect } from '@playwright/test'
+import { createDataset, loggedInUser } from './setup/utils'
+
+test.describe('search page', () => {
+    test.beforeEach(async ({ page, request, context }) => {
+        await loggedInUser({ page, request, context })
+    })
+
+    test.describe('can not find existing dataset', () => {
+        test('find existing dataset', async ({ page }) => {
+            await page.goto('/')
+
+            await page.fill('input[name="searchTerm"]', 'tests titles')
+
+            await page.click('button[type="submit"]')
+
+            await expect(page).toHaveURL('/sok?searchTerm=tests%20titles')
+            await expect(page.getByText('test title')).toHaveCount(0)
+        })
+    })
+
+    test.describe('can find existing dataset', () => {
```

Comment:
We could make this a little bit more sophisticated by extending the "create dataset" utility function to be able to take a specific title and set it.

I would like the test to be three cases here:

We create 3 datasets
"Test title - 1"
"Test title - 2"
"Some other title - 3"

One test searches for "Test title" and expects to see the first two in the result.

One test searches for "Some other" and expects to see one result.

One test that searches for "some term that matches nothing" and expects 0 results.